### PR TITLE
Update stream samples to latest model

### PR DESCRIPTION
**Pull request by sobychacko**
* Update Stream applicaiton Kafka samples to use the latest functional model
* Use the new test binder to test the components

Resolves https://github.com/spring-cloud/spring-cloud-dataflow-samples/issues/131

### CODE REVIEW

File: dataflow-website/stream-developer-guides/streams/standalone-stream-kafka/usage-detail-sender/src/main/java/io/spring/dataflow/sample/usagedetailsender/UsageDetailSender.java [hunk 1]
```diff
@@ -1,31 +1,25 @@
...
-@EnableBinding(Source.class)
+@Configuration
 public class UsageDetailSender {
 
-	@Autowired
-	private Source source;
-
 	private String[] users = {"user1", "user2", "user3", "user4", "user5"};
 
-	@Scheduled(fixedDelay = 1000)
```
Comment (sabbyanandan) [inline dataflow-website/stream-developer-guides/streams/standalone-stream-kafka/usage-detail-sender/src/main/java/io/spring/dataflow/sample/usagedetailsender/UsageDetailSender.java]: Without this, how are we going to repeat the same payload every 1s? Perhaps you're changing the behavior of the sample then?

Comment (sobychacko) [reply dataflow-website/stream-developer-guides/streams/standalone-stream-kafka/usage-detail-sender/src/main/java/io/spring/dataflow/sample/usagedetailsender/UsageDetailSender.java]: @sabbyanandan The behavior is not changed. With the new functional model of suppliers, you don't need to add the `Scheduled` annotation any more. By default, there is a poller that triggers the supplier every second. So, you will see the same behavior.

Comment (sabbyanandan) [reply dataflow-website/stream-developer-guides/streams/standalone-stream-kafka/usage-detail-sender/src/main/java/io/spring/dataflow/sample/usagedetailsender/UsageDetailSender.java]: Ah, thank you for the education.

**Comment by sabbyanandan** [comment]
@sobychacko: Let's go!

**Pull request merged**